projects/iniparser/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Google LLC
+# Copyright 2025 Lars Möllendorf <lars@moellendorf.eu>
```

Comment:
Do you mind removing this? I don't it's against the rules, but it's better for me if I don't need to find out if this specifically is allowed
:-)